### PR TITLE
regexp-based inlines and callbacks

### PR DESCRIPTION
**Pull request by akolechkin**
Hi, 
I would like to share my extension for aiotg I've been using in my projects. You can take a look at ext.py file, I just copied from my project. It adds ability to handle regexp based callback and inlines based on callback_data and query. It's similar to regex command interface. If you think this behavior fits well, I can merge the extension into existing codebase. I created it as a subclass just because I wanted to have aiotg installed from pypi without forking it.

### CODE REVIEW

**Comment by szastupov** [comment]
Looks good, I'm definitely interested in merging it into bot.py (it shouldn't break existing API, though).

File: aiotg/ext.py [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+        """
+        def decorator(fn):
+            self.add_callback(regexp, fn)
+            return fn
+        return decorator
+
+    def add_inline(self, regexp, fn):
+        self._inlines.append((regexp, fn))
+
+    def inline(self, regexp):
```
Comment (szastupov) [inline aiotg/ext.py]: I like it but it breaks the existing api. Perhaps we can check arguments here and act differently when function or string is passed.

**Comment by akolechkin** [comment]
Yes, you're right. It's not a big deal. I've added one more check to the callback and inline regexp argument and merged ext.py into bot.py.

**Comment by szastupov** [comment]
Good job! Thanks a lot!

**Pull request merged**